packages/experimental/NativeButton/src/NativeButton.tsx

```diff
@@ -14,9 +14,9 @@ export const NativeButton = compose<NativeButtonType>({
     root: buildProps(
       (tokens, theme) => ({
         style: {
-          height: 35,
+          height: 36,
```

Comment:
For things like this, what prompted this slight tweak?

Reply:
I wanted to align to a 4 px grid, but this is minor and can be removed and done separately. 
We probably shouldn't hardcode the height and width in the style but I'm not a flexbox expert enough to know what to actually do so that this view just uses the intrinsicContentSize